jekyll/_cci2/concurrency.adoc

```diff
@@ -0,0 +1,114 @@
+---
+version:
+- Cloud
+- Server 3
+- Server 2
+---
+= Concurrency
+:page-layout: classic-docs
+:page-liquid:
+:page-description: This documentation summarizes concurrency, and suggests ways to use concurrency to your advantage.
+:icons: font
+:toc: macro
+:toc-title:
+
+This documentation summarizes concurrency, and suggests ways to use concurrency to your advantage. 
+
+In CircleCI, **concurrency** refers to utilizing multiple containers to run multiple builds at the same time. This is different from CircleCI **parallelism**, which is test-splitting across multiple containers. If you would like information on parallelism, visit the <<parallelism-faster-jobs/#,Running Tests in Parallel>> page.
+
+Every plan with CircleCI includes the ability to concurrently run jobs. However, the number of jobs you can run concurrently differs between plans. Please view the https://circleci.com/pricing/[pricing page] to see up to date information on the number of concurrent job runs included in your plan.
+
+toc::[]
+
+== Concurrency
+Concurrency allows multiple jobs to be run at the same time in a workflow through the use of multiple containers. To keep the system stable for all CircleCI customers, we implement different soft concurrency limits on each of the <<configuration-reference/#resourceclass,resource classes>> for different executors. 
+
+If jobs are queued, it is possible you are hitting these limits. Hitting your limit may depend on a few factors, such as how many people in your organization are running jobs at once, or if you have overlapping workflows.
+
+Customers on a Performance or Scale plan can request an increase to those limits at no extra charge by contacting https://support.circleci.com/hc/en-us/requests/new[CircleCI support].
+
+=== Concurrency in workflows
+To run a set of concurrent jobs, you will need to add a new `workflows` section to the end of your existing `.circleci/config.yml` file. With CircleCI `version: 2.1`, you do not need to specify a `version` in the new `workflows` section. If you are still using CircleCI `version: 2`, you will need to specify the `version` in the `workflows` section.
+
+The simple example below shows the default workflow orchestration with two concurrent jobs. The `workflows` key needs to have a unique name. In this example, the unique name is `build_and_test`. The `jobs` key is nested under the uniquely named workflow, and contains the list of job names. Since the jobs have no dependencies, they will run concurrently.
+
+```yaml
```

Comment:
Due to the small number of people still on version 2 I think it's safe to remove references to the workflows version to avoid confusing begginer Cloud users, and simply have a separate version 2 example for server customers under the main example - then when we have tabs available in asciidoc we can use Cloud, server 3 and server 2 tabs here as we do elsewhere?